llvm_util/llvm2alive.cpp

```diff
@@ -692,15 +692,29 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
 
   RetTy visitUnreachableInst(llvm::UnreachableInst &i) {
     auto fals = get_operand(llvm::ConstantInt::getFalse(i.getContext()));
-    return make_unique<Assume>(*fals, /*if_non_poison=*/false);
+    return make_unique<Assume>(*fals, Assume::AndNonPoison);
   }
 
   RetTy visitIntrinsicInst(llvm::IntrinsicInst &i) {
     switch (i.getIntrinsicID()) {
     case llvm::Intrinsic::assume:
     {
+      unsigned n = i.getNumOperandBundles();
+      for (unsigned j = 0; j < n; ++j) {
+        auto bundle = i.getOperandBundleAt(j);
+        llvm::StringRef name = bundle.getTagName();
+        if (name == "noundef") {
```

Comment:
should we take this opportunity to skip unknown bundle tags to avoid false-positives?
We have a few unit tests that fail because of unhandled bundle tags.

Reply:
Yep, I'll update it